### DISCUSSION TITLE
Disable right arrow displayed in prompt when switching directories

### DISCUSSION THREAD

**Discussion by MattNguyen** [ohmyzsh/ohmyzsh#11688] [Help] [answered]
How can I disable the right arrow prefix at the beginning of the prompt? It seems to only be displayed when switching directories.

<img width="311" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/1606774/ddea0d2c-b6c8-416a-a572-d0c50da32678">

**Comment by rwmitchell** [comment, marked as answer]
You’re using iTerm, that is an iTerm feature using shell integration. 

And it appears every time you enter a real command instead of just banging on Enter

**Reply by MattNguyen** [reply]
I see, I presumed it was OMZ since it would disappear when I removed OMZ from zshrc. Thanks!

For anyone else who runs into this and wants to disable the arrow, go to Preferences > Profiles > Terminal > Shell Integration and untick "Show mark indicators".

<img width="942" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/1606774/7b44eb56-ff4b-41c8-a822-9f263f164732">

**Comment by rwmitchell** [comment]
Have you looked to see what those indicators provides and why you might actually want them?

It allows you to use hotkeys to page backward and forward thru the screen output jumping to each prompt, a very quick way to find the beginning of a large command output.

**Reply by MattNguyen** [reply]
That does seem convenient. I usually use search within a tmux pane for the previous command used, visual yank, and `pbpaste | less` to view large output. I'm pretty comfortable with this workflow at this point, but I'll look into setting up hotkeys using this visual indicator. Right now, the uneven spacing between the indicator and my prompt gives me design anxiety, so I'll just disable it for the time being. Thanks again!

**Reply by MattNguyen** [reply]
If you have a visual explainer for how you use the indicator, I'd love to see it!

**Reply by rwmitchell** [reply]
```[CMD][SHIFT][Up-Arrow]``` and ```[CMD][SHIFT][Dn-Arrow]``` works for me and I'm pretty sure that is the default.  Seems pretty simple, not sure what you need more than just trying it.

**Reply by MattNguyen** [reply]
I see! Not very helpful for me since I use tmux with multiple panes within iterm2. If I use that shortcut, iterm2 will search through the window history rather than the current pane within tmux session, and I end up in a weird place! Thanks for the tip though!

**Reply by rwmitchell** [reply]
I'm not currently using tmux (have in the past and thought shell integration still worked), but I'd be surprised if Nachman didn't account for it.
